utils/prof_utils.py

```diff
@@ -1181,10 +1189,10 @@ def auto_gfit(profile, max_N=6, plot_name=None, ignore_threshold=None, min_comp_
         except(LittleClipError, LargeClipError, NoComponentsError) as e:
             logger.setLevel(loglvl)
             logger.info(e)
-            logger.info("Skipping slpha value: {}".format(alpha))
+            logger.info("Skipping alpha value: {}".format(alpha))
             logger.setLevel(logging.WARNING) #squelch logging for the loop
-
-
+    logger.setLevel(loglvl)
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=5062980)